sites/website/src/docs/design-systems/using-the-design-system.md

```diff
@@ -0,0 +1,130 @@
+---
+id: using-the-design-system
+title: Using the FAST Design System 
+sidebar_label: Using the Design System
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/using-the-design-system.md
+---
+
+This section goes into further detail on how the Design System can be used. It will use [FAST Frame](/docs/design-systems/fast-frame) Design System in the examples but none are *specific* to [FAST Frame](/docs/design-systems/fast-frame); use these features with your own Design System if you're not using [FAST Frame](/docs/design-systems/fast-frame).
```

Comment:
I feel like the first sentence is given already in the title.  Would suggest removing

Reply:
Maybe could begin with "Using FAST Frame in the Design system" or some variation perhaps

Reply:
I added this to be sure folks didn't conflate this with documentation that is specific to FAST Frame because it is general purpose documentation. I think removing this would cause confusion between using the FAST Frame Design System and using *any* Design System